src/components/Earn/AdditionalOpportunities/useOpportunities.tsx

```diff
@@ -0,0 +1,272 @@
+import umamiIcon from "img/ic_umami.svg";
+import venusIcon from "img/ic_venus.svg";
+
+export type OpportunityTag =
+  | "lending-and-borrowing"
+  | "looping"
+  | "delta-neutral-vaults"
+  | "autocompound"
+  | "yield-trading";
+
+export type Opportunity = {
+  name: string;
+  description: ReactNode;
+  tags: OpportunityTag[];
+  assets: OpportunityAsset[];
+  icon: string;
+  link: string;
+};
+
+export type OpportunityAsset = { type: "market" | "glv" | "token"; address: string } | { type: "stGmx" };
+
+export const ST_GMX_OPPORTUNITY_ASSET: OpportunityAsset = { type: "stGmx" };
+
+const makeMarketAsset = (address: string): OpportunityAsset => ({ type: "market", address });
+const makeGlvAsset = (address: string): OpportunityAsset => ({ type: "glv", address });
+const makeTokenAsset = (address: string): OpportunityAsset => ({ type: "token", address });
+
+export const getOpportunityAssetKey = (asset: OpportunityAsset): string => {
+  if (asset.type === "stGmx") {
+    return "stGmx";
+  }
+  return asset.type + ":" + asset.address;
+};
+
+export const getOpportunityAssetLabel = (
+  asset: OpportunityAsset,
+  {
+    marketsInfoData,
+    tokensData,
+  }: {
+    marketsInfoData: GlvAndGmMarketsInfoData | undefined;
+    tokensData: TokensData | undefined;
+  }
+): string | undefined => {
+  switch (asset.type) {
+    case "stGmx":
+      return "Staked GMX";
+    case "token": {
+      const token = tokensData?.[asset.address];
+      return token?.symbol;
+    }
+    case "market": {
+      const marketInfo = marketsInfoData?.[asset.address];
+      if (marketInfo) {
+        return `GM: ${getMarketIndexName(marketInfo)} [${getMarketPoolName(marketInfo)}]`;
+      }
+      return undefined;
+    }
+    case "glv": {
+      const glvInfo = marketsInfoData?.[asset.address];
+      if (glvInfo) {
+        const poolName = getMarketPoolName(glvInfo);
+        return `GLV [${poolName}]`;
+      }
+      return undefined;
+    }
+    default:
+      mustNeverExist(asset);
+  }
+};
+
+export const useOpportunityTagLabels = () => {
+  return {
+    "lending-and-borrowing": t`Lending and Borrowing`,
+    looping: t`Looping`,
+    "delta-neutral-vaults": t`Delta Neutral Vaults`,
+    autocompound: t`Autocompound`,
+    "yield-trading": t`Yield Trading`,
+  };
+};
+
+export const useOpportunities = () => {
+  const { chainId } = useChainId();
+
+  const opportunities: Partial<Record<ContractsChainId, Opportunity[]>> = useMemo(() => {
+    return {
+      [ARBITRUM]: [
+        {
+          name: "Dolomite",
+          icon: dolomiteIcon,
+          description: (
+            <Trans>
+              Lend out your GLV or GM tokens, and borrow against them, or put your LP tokens into strategies to maximize
+              your yield.
+            </Trans>
+          ),
+          tags: ["lending-and-borrowing", "looping"],
+          assets: [
+            // GLV BTC [WBTC-USDC]
+            makeGlvAsset("0x528A5bac7E746C9A509A1f4F6dF58A03d44279F9"),
```

Comment:
let's move all addresses to configs please

For market addresses would be good to have an util to get address by market name from sdk/configs/markets

I know it's not about this task, but having one more place with market addresses and comment names not really convenient